### PR TITLE
Add theorem 275: lots + connected + separable => second countable. Sp…

### PR DESCRIPTION
**Pull request by keithpenrod**
1. Add theorem 275: lots + connected + separable => second countable
2. Add property 133 (lots) to Space 93 (weak parallel lines) to give an example indicating that connected is a necessary hypothesis for T275

### CODE REVIEW

File: theorems/T000275.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+if:
+  and:
+  - P000133: true
+  - P000036: true
+  - P000026: true
+then:
+  P000027: true
+---
+
+Let $X$ be a connected linearly ordered topological space with countable dense subset $S$. Then the collection of all open intervals and rays of the form $(a, b)$ with $a,b\in S\cup \{-\infty, \infty\}$ forms a countable base for $X$.
```
Comment (prabau) [inline theorems/T000275.md:12]: We should add how the connected hypothesis is used in the argument.  Also need some reference.

**Comment by prabau** [comment]
Good example showing the connectedness is needed.

Now for references for the proposed theorem, I found these two for example (really the same crossposted by the same asker):
https://mathoverflow.net/questions/351953
https://math.stackexchange.com/questions/3534793

This shows that

> (LOTS + connected + separable ==> metrizable)   (*)

(via showing that the resulting space is second countable).  But that is a more general result that would imply the proposed T275, since (metrizable + separable ==> second countable).
Do you guys think it would make sense to have (*) instead?

File: theorems/T000275.md (hunk 1 above)
Comment (StevenClontz) [reply theorems/T000275.md]: See [72d5867](https://github.com/pi-base/data/pull/147/commits/72d586700b2e467cff948c839f29d0047e8004aa) for my addition on connecteness.

**Comment by keithpenrod** [comment]
I'm fine with either result (metrizable or second countable). I'm not sure that I like the use of Q to denote the dense set, just because it might be confused with the mathbb Q.

**Comment by StevenClontz** [comment]
I was being somewhat cheeky: every connected separable LOTS is homeomorphic to a subinterval of the reals, so Q really is the rationals in a certain sense...

But as the contributor you should feel free to revert it back to S if you prefer.

**Comment by keithpenrod** [comment]
Just querying the site as it is, (lots + metrizable + ~second countable) is possible with the discrete topology on R, but (lots + second countable + ~metrizable) is impossible by Theorems 23, 90, 112, 113, 115, 273 (basically T5 => T3, second countable => sigma-finite local base, T3+sigma-finite local base => metrizable). So, in that sense second countable is the sharper result for LOTS.

**Comment by prabau** [comment]
> I'm inclined to add the theorem as-is; metrizable and second-countable generally are orthogonal properties, but here are equivalent due to assumptions guaranteeing T3+separable, so let's add what matches the reference.

Yeah, I think that makes sense.  The result that implies second countable is simpler and is equivalent to the one that implies metrizable.  So better go with the simpler one.

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**